### PR TITLE
Nomi/minor improvements

### PR DESCRIPTION
**Pull request by mechanomi**
this PR adds changes to address open issues with the "scope: minor" label

parent issue: https://github.com/crate/tech-writing-domain/issues/384

see commit messages for details

### CODE REVIEW

**Comment by amotl** [comment]
Thanks Naomi. I will add some minor comments, but also would like to ask @mfussenegger or @seut for an additional review.

File: docs/general/ddl/sharding.rst [hunk 1]
```diff
@@ -62,11 +62,18 @@ is applied, (see :ref:`ref_clustered_clause`).
...
 
-The column used for routing can be freely defined using the ``CLUSTERED BY
-(<column>)`` statement and is used to route a row to a particular shard.
+*shard number = hash(routing column) % total primary shards*
+
+When hash values are distributed evenly (which will be approximately true in
+most cases) rows will be distributed evenly amongst the fixed amount of
+available shards.
+
+The routing column can be specified defined with the ``CLUSTERED BY
```
Comment (amotl) [inline docs/general/ddl/sharding.rst:74]:
```suggestion
The routing column can be defined with the ``CLUSTERED BY
```

File: docs/general/dql/selects.rst [hunk 2]
```diff
@@ -525,6 +452,45 @@ does always return ``NULL`` when comparing ``NULL``.
...
+CrateDB supports a variety of :ref:`array comparisons <sql_array_comparisons>`.
+
+
+.. _sql_dql_in:
+
+``IN``
+------
+
+CrateDB also supports the binary operator ``IN``, which allows you to verify
+the membership of left-hand operand in a right-hand set of expressions. Returns
```
Comment (amotl) [inline docs/general/dql/selects.rst:469]:
```suggestion
the membership of the left-hand operand in a right-hand set of expressions. Returns
```

File: docs/sql/statements/create-table.rst [hunk 3]
```diff
@@ -190,9 +190,10 @@ The optional CLUSTERED clause specifies how a table should be distributed
 accross a cluster.
 
 :num_shards:
-  Specifies the number of shards a table is stored in. Must be greater
-  than 0. If not provided the number of shards is calculated based on
-  the number of currently active data nodes with the following formula::
+  Specifies the number of :ref:`shards <sql_ddl_sharding>` a table is stored
+  in. Must be greater than 0. If not provided the number of shards is
```
Comment (amotl) [inline docs/sql/statements/create-table.rst:194]:
```suggestion
  in. Must be greater than 0. If not provided, the number of shards is
```

File: docs/sql/statements/create-table.rst [hunk 4]
```diff
@@ -208,13 +209,19 @@ accross a cluster.
...
+
+
 .. _partitioned_by_clause:
 
 ``PARTITIONED BY``
 ==================
 
-The PARTITIONED clause splits the created table into separate partitions for
-every distinct combination of values in the listed columns.
+The PARTITIONED clause splits the created table into separate :ref:`partitions
```
Comment (amotl) [inline docs/sql/statements/create-table.rst:222]:
```suggestion
The ``PARTITIONED`` clause splits the created table into separate :ref:`partitions
```

**Review by amotl — APPROVED** [review]
Great overhaul! 💯

File: docs/general/ddl/sharding.rst (hunk 1 above)
Comment (mechanomi) [reply docs/general/ddl/sharding.rst:74]: I don't think "defined" is correct here

here's my understanding of the two words in a technical context:

- "define" - describe a specific noun (i.e., describe what it is)
- "specify" - highlight/choose a specific noun for some purpose (i.e., something that has been defined elsewhere)

in this case, the column is *defined* in the table definition. its definition is `first_column integer` (for example). the `CLUSTERED BY` clause *specifies* that the (pre-defined) `first_column` column is to be used for shard routing

**Comment by mechanomi** [comment]
@amotl thank you for the detailed review! I have pushed some fixups

File: docs/general/ddl/sharding.rst (hunk 1 above)
Comment (amotl) [reply docs/general/ddl/sharding.rst:74]: Sure. Then delete `defined`.

```suggestion
The routing column can be specified with the ``CLUSTERED BY
```

Comment (mechanomi) [reply docs/general/ddl/sharding.rst:74]: omg I totally missed that 😅 thanks!

**Pull request merged**